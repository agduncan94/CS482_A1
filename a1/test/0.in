>header seq1
ACTGACTGTCGATCGAGGAGAGA

ACTCAGAAGTT

ACTGT
