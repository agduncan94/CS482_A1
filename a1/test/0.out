AAA 0.0000
AAC 0.0270
AAG 0.0270
AAT 0.0000
ACA 0.0000
ACC 0.0000
ACG 0.0000
ACT 0.1081
AGA 0.0811
AGC 0.0000
AGG 0.0270
AGT 0.0270
ATA 0.0000
ATC 0.0270
ATG 0.0000
ATT 0.0000
CAA 0.0000
CAC 0.0000
CAG 0.0270
CAT 0.0000
CCA 0.0000
CCC 0.0000
CCG 0.0000
CCT 0.0000
CGA 0.0541
CGC 0.0000
CGG 0.0000
CGT 0.0000
CTA 0.0000
CTC 0.0270
CTG 0.0811
CTT 0.0000
GAA 0.0541
GAC 0.0270
GAG 0.0811
GAT 0.0270
GCA 0.0000
GCC 0.0000
GCG 0.0000
GCT 0.0000
GGA 0.0270
GGC 0.0000
GGG 0.0000
GGT 0.0000
GTA 0.0000
GTC 0.0270
GTG 0.0000
GTT 0.0270
TAA 0.0000
TAC 0.0270
TAG 0.0000
TAT 0.0000
TCA 0.0270
TCC 0.0000
TCG 0.0541
TCT 0.0000
TGA 0.0270
TGC 0.0000
TGG 0.0000
TGT 0.0541
TTA 0.0270
TTC 0.0000
TTG 0.0000
TTT 0.0000
